genkit/prompts/textRefinement.prompt

```diff
@@ -53,4 +53,9 @@ Example Output:
 Input:
 {{request}}
 
+Output must be in JSON format and follow the following schema:
```

Comment:
Same here, let's remove the output schema definition from the definition and just leave it here in the prompt.